spec/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -565,15 +571,14 @@ The IBC handler performs the following steps in order:
 - Checks that the packet sequence is the next sequence the channel end expects to receive (for ordered channels)
 - Checks that the timeout height has not yet passed
 - Checks the inclusion proof of packet data commitment in the outgoing chain's state
-- Sets the opaque acknowledgement value at a store path unique to the packet (if the acknowledgement is non-empty or the channel is unordered)
+- Sets a store path to indicate that the packet has been received (unordered channels only)
```

Comment:
Don't we need to set recvPacket all the time now? As we do not rely on acknowledgements?

If a synchronous acknowledgement obviates the need for `recvPacket` then this should specify that either acknowledgement or receipt must be written atomically with `onRecv`.

Or maybe you can explain why I misunderstand this.

Reply:
`recvPacket` always needs to be called, as usual, regardless of whether or not an acknowledgement will be written.